resources/Dockerfile

```diff
@@ -91,3 +91,20 @@ RUN export ARCH="$(uname -m)" \
 RUN echo 'source $CARGO_HOME/env' >> $HOME/.bashrc \
     && mkdir $HOME/.cargo \
     && ln -s $CARGO_HOME/env $HOME/.cargo/env
+
+# install virtiofsd
+RUN git clone --depth 1 https://gitlab.com/virtio-fs/qemu.git -b qemu5.0-virtiofs-dax \
+    && cd qemu \
+    && ./configure --prefix=$PWD --target-list=x86_64-softmmu \
+    && make virtiofsd -j `nproc` \
+    && cp virtiofsd /usr/local/bin \
+    && cd .. && rm -rf qemu
+
+# install virtiofsd-rs
+RUN git clone --depth 1 https://gitlab.com/virtio-fs/virtiofsd-rs.git \
```

Comment:
For `virtiofsd` the approach is fine since we are always relying on the same tag.
But for `virtiofsd-rs`, we're modifying our CI behavior a bit since we're gonna be stuck on the version at the time the Docker image is created. How are we gonna be able to update it from time to time?
/cc @rbradford

Reply:
Maybe leave virtiofsd-rs as it is currently.

Reply:
Sounds good. Wasn't aware we are intentionally relying on the latest version of virtiofsd-rs. Will fix it in the separate PR.